runtime/src/lib.rs

```diff
@@ -47,7 +47,7 @@ impl sdk::Runtime for Runtime {
     const VERSION: Version = sdk::version_from_cargo!();
     /// Current version of the global state (e.g. parameters). Any parameter updates should bump
     /// this version in order for the migrations to be executed.
-    const STATE_VERSION: u32 = 5;
+    const STATE_VERSION: u32 = 6;
```

Comment:
how does this change work? no migrations, only bump to prevent downgrading?

Reply:
Bump to trigger a migration that will update the parameters.

Reply:
oh right that new gas parameter